x-pack/plugins/canvas/public/components/fullscreen/fullscreen.scss

```diff
@@ -11,15 +11,22 @@ body.canvas-isFullscreen {
   }
 
   // hide all the interface parts
-  nav.global-nav,
+  nav.global-nav, // K6 global side nav
+  .header-global-wrapper, // K7 global top nav
   .canvasLayout__stageHeader,
   .canvasLayout__sidebar,
   .canvasLayout__footer,
   .canvasGrid {
     display: none;
   }
 
+  // remove space for K7 global top nav
+  .header-global-wrapper + .app-wrapper {
+    top: 0;
+  }
+
   .canvasLayout__stageContentOverflow {
+    display: block; // fixes center alignment for Safari
     overflow: visible;
     position: static;
     top: auto;
```

Comment:
Not as part of this PR, but maybe `top/left/bottom/right` could be deleted, looks like they aren't being used: https://developer.mozilla.org/en-US/docs/Web/CSS/position#Values